src/constants/languages.tsx

```diff
@@ -14,76 +14,89 @@ import {
 } from '@deriv/quill-icons';
 
 export const LANGUAGES = [
-    {
-        code: 'AR',
-        displayName: 'العربية',
-        icon: <FlagUnitedArabEmiratesIcon height={24} width={36} />,
-        placeholderIcon: <FlagUnitedArabEmiratesIcon height={12} width={18} />,
-    },
-    {
-        code: 'DE',
-        displayName: 'Deutsch',
-        icon: <FlagGermanyIcon height={24} width={36} />,
-        placeholderIcon: <FlagGermanyIcon height={12} width={18} />,
-    },
     {
         code: 'EN',
         displayName: 'English',
         icon: <FlagUnitedKingdomIcon height={24} width={36} />,
         placeholderIcon: <FlagUnitedKingdomIcon height={12} width={18} />,
+        placeholderIconInMobile: <FlagUnitedKingdomIcon height={14.67} width={22} />,
+    },
+    {
+        code: 'AR',
+        displayName: 'العربية',
+        icon: <FlagUnitedArabEmiratesIcon height={24} width={36} />,
+        placeholderIcon: <FlagUnitedArabEmiratesIcon height={12} width={18} />,
+        placeholderIconInMobile: <FlagUnitedArabEmiratesIcon height={14.67} width={22} />,
```

Comment:
you should replace this with `FlagArabLeagueIcon`

Reply:
Thanks @matin-deriv will change that 🙏